readme.md

```diff
@@ -27,23 +29,32 @@ Návod je určený pro Windows, v Linuxu by mělo stačit nainstalovat všechno
   - Vytvořit si ssh klíč – spusťte git gui (přidalo se vám do startu) a v něm pak help > show SSH key > generate key > copy to clipboard.
   - SSH klíč si spárovat s githubem – vpravo nahoře vaše ikona > settings > SSH and GPG keys > new SSH key. Do políčka Key vložíte, co jste vykopírovali z git gui. Title je jedno, to slouží jen jako popis pro vás, kdybyste klíčů měli víc.
 - Vytvořit databázi
-  - Nejdřív je potřeba sehnat si od někoho zálohu DB
-  - Otevřít si http://localhost/adminer, přihlásit se jako root/(žádné heslo)
+  - Nejdřív je potřeba sehnat si od někoho zálohu DB (opět IT tým)
+  - Nastavit heslo "root" pro uživatele root pro MySQL (Adminer (níže) nedovoluje přihlašování bez hesla)
```

Comment:
Hodil bych sem info, že to heslo může být vlastně jakékoliv - nováčci mají tendenci vzývat dokumentaci jako božstvo a věřit všemu doslovně - tak abychom je nedrželi v mylné víře:
```suggestion
  - Nastavit heslo, třeba "root", pro uživatele root pro MySQL (Adminer (níže) nedovoluje přihlašování bez hesla)
```